app/javascript/stylesheets/_base_layout.scss

```diff
@@ -365,21 +383,25 @@ a {
 			background-clip: padding-box;
 			font-size: .95em;
 			li {
-				margin: 1rem 0;
+				margin: 0.1rem 0;
 			}
 		}
 		a, .nav-link {
 			&:hover {
 				background-color: transparent;
-				color: $link-color;
+				color: map-get($theme-text-colors, 'secondary');
 			}
 			&.active {
 				background-color: transparent;
 			}
 		}
+    .nav-link {
+      border-radius:0;
+    }
 		li {
 			a:hover {
 				@extend .swatch-info;
+        color: map-get($theme-text-colors, 'primary');
```

Comment:
this change codifies an expectation that the `primary` text color also forms a high-contrast pair with the `info` background color - seems dicey? Maybe not, but would want to document.

Reply:
also good catch. same comment as above https://github.com/cul/ldpd-dcv/pull/176#:~:text=er%2Dk-,16%20seconds%20ago,-good%20catch.%20i%27m